lib/index.js

```diff
@@ -18,20 +18,50 @@ var Appboy = module.exports = integration('Appboy')
   .global('appboy')
   .option('apiKey', '')
   .option('safariWebsitePushId', '')
+  .option('allowCrawlerActivity', false)
+  .option('doNotLoadFontAwesome', false)
```

Comment:
Nit: I realize you're just copying over Braze's stuff, but maybe we can avoid the double-negatives by making this setting be called "LoadFontAwesome"?

Reply:
Mm, I see what you're saying. Better to have the setting how up with a bunch of things grayed out than have them activated by default?

Reply:
Correct.